athena-gcs/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>aws-athena-query-federation</artifactId>
+        <groupId>com.amazonaws</groupId>
+        <version>2022.42.2</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>athena-gcs</artifactId>
+    <version>2022.42.2</version>
+
+    <properties>
+        <slf4jVersion>1.7.30</slf4jVersion>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>athena-federation-integ-test</artifactId>
+            <version>2022.42.2</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
+            </exclusions>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.arrow</groupId>
+            <artifactId>arrow-vector</artifactId>
+            <version>3.0.0</version>

```

Comment:
You should not be using 3.0.0 anymore. The rest of the federation github repository has already been upgraded to using 9.0.0.

Please make use of the ${apache.arrow.version} property as well.

Reply:
Used ${apache.arrow.version} instead